### PR TITLE
Add network-rule support for Cognitive Services

### PR DESCRIPTION
**Pull request by yangyuan**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by tjprescott** [comment]
@MyronFanQiu evaluate this PR against the guidance here that we discussed yesterday:
https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md#network-rule-commands

File: src/azure-cli/azure/cli/command_modules/cognitiveservices/custom.py [hunk 1]
```diff
@@ -79,24 +81,104 @@ def create(
...
+        rules.ip_rules.append(IpRule(value=ip_address))
+
+    return client.update(resource_group_name, account_name, properties={"networkAcls": rules})
+
+
+def remove_network_rule(cmd, client, resource_group_name, account_name, ip_address=None, subnet=None,
+                        vnet_name=None):  # pylint: disable=unused-argument
+    sa = client.get_properties(resource_group_name, account_name)
+    rules = sa.network_acls
+    if rules is None:
```
Comment (MyronFanQiu) [inline src/azure-cli/azure/cli/command_modules/cognitiveservices/custom.py]: could we just return?

Comment (yangyuan) [reply src/azure-cli/azure/cli/command_modules/cognitiveservices/custom.py]: Yes sure.

**Comment by yangyuan** [comment]
@MyronFanQiu conflict solved.

For the `default_action` filed.

We want to provide similar experiences with Azure Storage:
https://docs.microsoft.com/en-us/cli/azure/storage/account/network-rule?view=azure-cli-latest
So in CLI we don't support modifying default_action (which is deny by default).

And we don't have `action` filed in rules. (you can consider it as "Allow" by default).

**Comment by MyronFanQiu** [comment]
@yangyuan Got it. Please fix the Lint and Style errors. You can use `azdev style <service_name` to check it locally.

**Comment by yangyuan** [comment]
@MyronFanQiu all checks passed :)

**Pull request merged**